### ISSUE TITLE
Pagination doesn't work after search

### ISSUE THREAD

**Issue by yusupprog** [ratiw/vuetable-2#252]
Hi, when i'm searching and results are less than per page, pagination component being removed from page and then when i'm searching again with empty string it throws error
```
Error in event handler for "vuetable:pagination-data": "TypeError: Cannot read property 'setPaginationData' of undefined"

```
because it can't find `this.$refs.pagination`e because it has been removed after first search.
Tried to wrap 

```
<div v-if="tablePagination && tablePagination.last_page > 1" :class="css.wrapperClass">
    <a @click="loadPage(1)"
      :class="['btn-nav', css.linkClass, isOnFirstPage ? css.disabledClass : '']">
        <i v-if="css.icons.first != ''" :class="[css.icons.first]"></i>
        <span v-else>&laquo;</span>
    </a>
...
```
with another div and it works.

is it a bug?

**Comment by ratiw** [comment]
I'm not sure if it's a bug. But I haven't encountered this behavior. Last time I checked, everything seems fine to me. Are you using vue-router? If so, that might be the problem.

**Comment by yusupprog** [comment]
Yes, I'm using vue-router but error occurs without changing route

**Comment by ratiw** [comment]
Try to inspect the search result that returned from the server and if the returned result is correct or not. The only thing I can think of is that there's a problem with the returned result.

You can also clone this repo, then run `npm install && npm run dev`. Play with the sameple, test the search that has only one page result, then try again to search with empty string. You can then look at `main.js` to see what's different in your implementation.

**Comment by lfserrra** [comment]
Estou com o mesmo problema, quando filtro de alguma forma que não é necessário a página e depois quando volta com uma quantidade que é possível paginar ele da esse problema.

Pode ser alguma outra biblioteca ou o próprio vue?

Aqui eu dei um npm update e ocorreu esse problema.

**Comment by lfserrra** [comment]
I'm having the same problem, when I filter somehow that the page is not needed and then when it comes back with an amount that it's possible to paginate it out of that problem.

Could it be some other library or the vue itself?

Here I gave an npm update and this problem occurred.

**Comment by ratiw** [comment]
This sudden change in `v-if` behavior might related to vuejs/vue#6632. Will look into it and give an update later.

**Comment by ratiw** [comment]
This has been fixed in v1.6.6 of Vuetable-2. See the release note [here](https://github.com/ratiw/vuetable-2/releases/tag/v1.6.6)

**Comment by CrashLaker** [comment]
Hi.

Got the same issue using v1.7.6
i'm also using vue-router

had to
```javascript
onPaginationData(paginationData) {
            if (this.$refs.paginationTop)
                this.$refs.paginationTop.setPaginationData(paginationData);
            if (this.$refs.paginationInfoTop)
                this.$refs.paginationInfoTop.setPaginationData(paginationData);
            if (this.$refs.pagination)
                this.$refs.pagination.setPaginationData(paginationData);
            if (this.$refs.paginationInfo)
                this.$refs.paginationInfo.setPaginationData(paginationData);
        },
```
